### DISCUSSION TITLE
Is Linux supported?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [chocolatey/choco#3579] [Q&A] [answered]
I don't see any mention of non-Windows OSes.

**Comment by TheCakeIsNaOH** [comment, marked as answer]
It is supported for selected tasks:
https://docs.chocolatey.org/en-us/guides/non-windows/

In short, Linux support is for building packages, useful for developers that have build environments based on Linux. The day-to-day use of Chocolatey CLI for installing/upgrading/uninstalling software is Windows only.
